config/services.yaml

```
@@ -0,0 +1,11 @@
+services:
+    Oneup\ContaoSentryBundle\Twig\ContaoSentryTwigExtension:
+        tags:
+            - { name: twig.extension }
+
+    Oneup\ContaoSentryBundle\Twig\ContaoSentryTwigRuntime:
```

As of current best practices we should not use FQCN as service ids.
```suggestion
    oneup_contao_sentry.twig.runtime:
        class: Oneup\ContaoSentryBundle\Twig\ContaoSentryTwigRuntime
```